### DISCUSSION TITLE
Predicting Football Match Outcomes Using Bayesian Poisson-Gamma Distribution

### DISCUSSION THREAD

**Discussion by giscus** [quarto-dev/quarto-web#1267] [General]
# Predicting Football Match Outcomes Using Bayesian Poisson-Gamma Distribution

http://localhost:7319/

**Comment by cderv** [comment]
@OKcomputer626 Unfortunately, wrong config: Reported upstream already 
- https://github.com/OKcomputer626/Projects-and-Stuff/issues/1
